installer/client/cli/fabric.py

```diff
@@ -177,32 +177,32 @@ def main():
     else:
         text = standalone.get_cli_input()
     if args.stream and not args.context:
-        if args.remoteOllamaServer:
-            standalone.streamMessage(text, host=args.remoteOllamaServer)
+        if os.environ["REMOTE_OLLAMA_SERVER"] or args.remoteOllamaServer:
```

Comment:
Calling `os.environ["REMOTE_OLLAMA_SERVER"]` will result in a `KeyError: 'REMOTE_OLLAMA_SERVER'` when a user doesn't have the `REMOTE_OLLAMA_SERVER` key in their `.env` file.

You should be using something like `os.environ.get('REMOTE_OLLAMA_SERVER', None)` instead.

(This comment is applicable for all similar calls you added.)

Reply:
Agree, thanks! I replaced the `os.environ["REMOTE_OLLAMA_SERVER"]` calls with `os.environ.get('REMOTE_OLLAMA_SERVER', None)` as you suggested.

Additionally, I added all of that to the PraisonAI agents section, replacing the `http://localhost:11434/v1` to make agents call the remote Ollama server if needed.